content/news/2021-06-24.md

```diff
@@ -0,0 +1,23 @@
+---
+Title: "OpenHW TV S02 E06: CORE-V eXtension I/F"
```

Comment:
the page that this generates isn't a clean URL: https://deploy-preview-342--eclipsefdn-openhwgroup.netlify.app/news/2021/06/24/openhw-tv-s02-e06-core-v-extension-i/f/ see the `i/f/` at the end. Can you try adding a `slug` property with that slash removed? https://gohugo.io/content-management/urls/#permalink-configuration-values see `:slug`

Reply:
I saw there was already a ":slug" in the config.toml - [permalinks].
I removed the slash in the "Title" and it didn't impact the title for the tab and the header for the news, so I guess this solves the problem?